index.css

```diff
@@ -0,0 +1,142 @@
+body {
+  font-family: 'Poppins', sans-serif;
+  box-sizing: border-box;
+  font-style: normal;
+}
+
+.header-container {
+  background: url(./images/header-bg-img.svg) no-repeat;
+  background-size: 100% auto;
+}
+
+nav {
+  padding: 5% 10%;
+}
+
+nav > img:last-child {
+  align-self: center;
+  width: 35px;
+  height: 25px;
+  cursor: pointer;
+}
+
+#logo {
+  font-family: Impact, Haettenschweiler, 'Arial Narrow Bold', sans-serif;
+  word-spacing: 0;
+  font-weight: bolder;
+  font-size: 1.5rem;
+  text-decoration: none;
+  font-style: oblique;
+  color: #6070ff;
+}
+
+.headline {
+  margin: 0 12%;
```

Comment:
- [ Required ] Please consider giving the `.headline` section a margin of `24px` on the `margin-left` and `margin-right` as it is on the `Figma` template the current margin of `12%` is slightly big. Please see the screenshot below for more details on this and rectify it.

## Figma design
<img src="https://user-images.githubusercontent.com/16392046/219852524-e957ef8a-f1de-4303-9075-540accabaabe.png" width="375px" />

## Your design
<img src="https://user-images.githubusercontent.com/16392046/219852628-be16f0b6-8d7d-4367-a561-1ab29ab904d2.png" width="375px" />